ownCloud/Bookmarks/Issues/Issues Subclasses/ConnectionIssueViewController.swift

```diff
@@ -138,6 +138,18 @@ extension ConnectionIssueViewController: UITableViewDataSource {
 		return displayIssues?.displayIssues.count ?? 0
 	}
 
+	override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+		guard let displayIssues = displayIssues, displayIssues.displayIssues.count == 1 else {
+			return super.tableView(tableView, heightForHeaderInSection: section)
+		}
+
+		if displayIssues.displayIssues.first?.localizedTitle == headerTitle {
+			return 0
```

Comment:
… while this is only returned if `displayIssues.displayIssues.count == 1` and its title equals `headerTitle`. For better readability, you could fold that condition into the `guard` statement and simplify this method to:

```swift
guard let displayIssues = displayIssues, displayIssues.displayIssues.count == 1, displayIssues.displayIssues.first?.localizedTitle == headerTitle else {
    return super.tableView(tableView, heightForHeaderInSection: section)
}

return 0
```

Reply:
fixed